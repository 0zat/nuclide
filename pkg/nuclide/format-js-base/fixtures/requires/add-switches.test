'use strict';

function foo() {
  switch (one) {
    case two:
      break;
  }
}
