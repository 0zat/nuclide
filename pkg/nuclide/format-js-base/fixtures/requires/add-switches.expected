'use strict';

var one = require('one');
var two = require('two');

function foo() {
  switch (one) {
    case two:
      break;
  }
}
